src/PermissionManager.php

```diff
@@ -0,0 +1,79 @@
+<?php
+declare(strict_types=1);
+
+namespace LotGD\Core;
+
+/**
+ * Permissions can be managed with the PermissionManager.
+ * 
+ * The PermissionManager class provides methods to work with permissions. It can
+ * be used to create or delete permissions, to remove, allow or deny permissions
+ * to actors and to check whether an actor has a certain permission or if it is
+ * explicitely denied from him.
+ * 
+ * The wording used in this class is:
+ *  - allowed, the actor has a certain permission in the allowed state.
+ *  - denied, the actor has a certain permission in the denied state.
+ * 
+ * To make this more clear, the following table summarizes how different methods
+ * react.
+ * 
+ * Method
+ *             State: | Unset | Allowed | Denied
+ * -------------------+-------+---------+---------
+ * isAllowed          | False | True    | False
+ * isDenied           | False | False   | True
+ * hasPermissionSet   | False | True    | True
+ */
+class PermissionManager
+{
+    const Allowed = 1;
+    const Denied = -1;
+    
+    const Superuser = "lotgd/core/superuser";
+    const AddScenes = "lotgd/core/scene/add";
```

Comment:
I think these scene/character-based permissions should be added as part of the scene or character class.